consensus/state_sync/module.go

```diff
@@ -112,78 +83,6 @@ func (m *stateSync) GetModuleName() string {
 	return stateSyncModuleName
 }
 
-func (m *stateSync) IsServerModEnabled() bool {
-	return m.serverMode
-}
-
 func (m *stateSync) SetLogPrefix(logPrefix string) {
```

Comment:
I did a `cmd shift f` on your branch and got this:

<img width="312" alt="Screenshot 2023-04-10 at 3 23 45 PM" src="https://user-images.githubusercontent.com/1892194/231010397-5465c418-78fb-4c7d-b06b-a993004914bc.png">

1. Is this being used anywhere?
2. If not, remove the function and the local variable